sssom/util.py

```diff
@@ -826,13 +827,13 @@ def filter_out_prefixes(df: pd.DataFrame, filter_prefixes) -> pd.DataFrame:
     rows = []
     
     for index, row in df.iterrows():
-        # Get list of CURIEs from the 3 columns (_defining_features) for the row.
-        list_of_curies = [get_prefix_from_curie(row_values) for row_values in row[_defining_features]]
+        # Get list of CURIEs from the 3 columns (KEY_FEATURES) for the row.
+        prefixes = {get_prefix_from_curie(curies) for curies in row[KEY_FEATURES]}
         # Confirm if none of the 3 CURIEs in the list above appear in the filter_prefixes list.
         # If TRUE, append row.
-        if not any(curie in list_of_curies for curie in filter_prefixes):
+        if not any(prefix in prefixes for prefix in filter_prefixes):
             rows.append(row)
     if rows:
         return pd.DataFrame(rows)
```

Comment:
this one should also have `columns=KEY_FEATURES`, right?

Reply:
No, we want the entire row if none of the `prefixes` from `[KEY_FEATURES]` exist in `filter_prefixes`

Reply:
No, we want the entire row if none of the `prefixes` from `[KEY_FEATURES]` exist in `filter_prefixes`